discovery/dns/dns.go

```diff
@@ -211,8 +211,10 @@ func (d *Discovery) refreshOne(ctx context.Context, name string, ch chan<- *targ
 			target = hostPort(addr.A.String(), d.port)
 		case *dns.AAAA:
 			target = hostPort(addr.AAAA.String(), d.port)
+		case *dns.CNAME:
+			// Ignore to prevent warning message from default case.
```

Comment:
This comment is still not useful. I can already tell this from the code alone, instead explain why this is the right thing to do.

Reply:
I misunderstood what you meant by comment- I thought you were referring to the title of the PR. My mistake, I'll get it adjusted.